### DISCUSSION TITLE
the new achievements look childish and feel out of place

### DISCUSSION THREAD

**Discussion by wsurles-vamp** [community/community#55859] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The design of the achievements looks like its from a free iphone game for kids. not a professional coding tool. 
Everything on the app is white, gray, black. And then those are bright colors and just look really out of place. IMO

**Comment by sofiarivas** [comment]
You can hide them from your profile: 🔗[docs]( https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements)
